lib/iex/lib/iex/dochelp/elixir.ex

```diff
@@ -0,0 +1,120 @@
+defmodule IEx.DocHelp.Elixir do
+  @moduledoc """
+  This module duplicates the prior behaviour of the iex h
+  command using the documentation interface defined by `IEx.DocHelp`.
+
+  """
+
+  import IEx.DocHelp
+
+  @behaviour IEx.DocHelp
+
+  def documentation(module) do
+    case has_get_docs?(module) do
+      true -> get_doc(module)
+      _   -> {:unknown, [{inspect(module), ""}]}
+    end
+  end
+
+  def documentation(module, function) do
+    case has_get_docs?(module) do
+      true -> get_doc(module, function)
+      _   -> {:unknown, [{inspect(module), ""}]}
+    end
+  end
+
+  def documentation(module, function, arity) do
+    case has_get_docs?(module) do
+      true -> get_doc(module, function, arity)
+      _  -> {:unknown, [{inspect(module), ""}]}
+    end
+  end
+
+  defp get_doc(module) when is_atom(module) do
+    docs = Code.get_docs(module, :moduledoc)
+    case docs do
+      nil -> {:not_found, [{inspect(module), "No moduledocs found\n"}]}
+      _   -> {:found, [{inspect(module), elem(docs, 1)}]}
+    end
+  end
+
+  defp get_doc(module, function) when is_atom(module) and is_atom(function) do
+    docs = Code.get_docs(module, :docs)
+    case docs do
+      nil -> {:not_found, [{"#{inspect module}.#{function}", "No documentation for #{inspect module}.#{function} found\n"}]}
```

Comment:
I wrote one, but I'm not real happy with it. It feels sort of macro like, but that seems like
overkill for just one simple function. I did extract out some helper functions that have cleanup
up this code a lot.